examples/background_workflow_example/.gitignore

```diff
@@ -0,0 +1,56 @@
+# Autosave files
+*~
+
+# build
+[Oo]bj/

```

Comment:
I'm not sure what the top do lines do here, but curious...

Reply:
I didn't explicitly add this (just copy/pasted the file). I believe this is an expression for some files to exclude. `*~` should match any file name ending in `~`.